pkg/provisioningv2/rke2/planner/etcdcreate.go

```diff
@@ -108,6 +108,10 @@ func (e *etcdCreate) createPlan(controlPlane *rkev1.RKEControlPlane, snapshot *r
 }
 
 func (e *etcdCreate) Create(controlPlane *rkev1.RKEControlPlane, clusterPlan *plan.Plan) error {
+	if !Provisioned.IsTrue(controlPlane) && controlPlane.Status.ETCDSnapshotCreatePhase == "" {
```

Comment:
What would be the situation when controlPlane is not Provisioned, but `controlPlane.Status.ETCDSnapshotCreatePhase` is not empty and why do we want to proceed further?
(Not questioning this, just wondering for my understanding)

Reply:
Once a snapshot is created, the plan for the `controlPlane` will be updated, causing it to go into a non-provisioned state. In that case, we want to continue.

This change only stops the snapshot create from starting when the `controlPlane` is in a non-provisioned state.